demo/api/src/comet-config.json

```diff
@@ -6,7 +6,7 @@
     "dam": {
         "additionalMimeTypes": [],
         "uploadsMaxFileSize": 500,
-        "allowedImageSizes": [320, 420, 640, 750, 768, 828, 1024, 1080, 1200, 1920, 2048, 3840],
+        "allowedImageSizes": [320, 420, 640, 750, 768, 828, 1020, 1024, 1080, 1200, 1920, 2048, 3840],
```

Comment:
`1024` is probably not needed then... maybe even all other values above it. 

Since this is only used for this specific mail (as far as I know), maybe it makes sense to remove all values that are not common image sizes in the mail 🤔 

What do you think @raphaelblum?

Reply:
We don't have any other images in the demo. I guess the allowedImageSizes were copied at project setup.

> maybe it makes sense to remove all values that are not common image sizes in the mai

Yes, I think it would make sense to remove all image sizes that are not common for mails. I don't think it makes sense having so much different imageSize options here.